### PR TITLE
Fixed bugs

### PR DESCRIPTION
**Pull request by songivan00**
- Change naming conventions of parser
- Change minor errors in the UG
- Change MESSAGE_USAGE of commands

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/MarkTaskCommand.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public class MarkTaskCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Marks the task identified by the index number used in the displayed task list as completed.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Parameters: {task_index (must be a positive integer)\n"
```
Comment (angkl0) [inline src/main/java/seedu/address/logic/commands/MarkTaskCommand.java:22]: Is the ending brace for task_index missing?

File: src/main/java/seedu/address/logic/commands/UnmarkTaskCommand.java [hunk 2]
```diff
@@ -19,7 +19,7 @@ public class UnmarkTaskCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Unmarks the task identified by the index number used in the displayed task list as not complete.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Parameters: {task_index (must be a positive integer)\n"
```
Comment (angkl0) [inline src/main/java/seedu/address/logic/commands/UnmarkTaskCommand.java:22]: Same as above

File: src/main/java/seedu/address/logic/commands/MarkTaskCommand.java (hunk 1 above)
Comment (songivan00) [inline src/main/java/seedu/address/logic/commands/MarkTaskCommand.java:22]:
```suggestion
            + "Parameters: {task_index} (must be a positive integer)\n"
```

File: src/main/java/seedu/address/logic/commands/UnmarkTaskCommand.java (hunk 2 above)
Comment (songivan00) [inline src/main/java/seedu/address/logic/commands/UnmarkTaskCommand.java:22]:
```suggestion
            + "Parameters: {task_index} (must be a positive integer)\n"
```

**Review by angkl0 — APPROVED** [review]
LGTM!

**Pull request merged**